<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>571</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>574</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Язык</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="actionEnglish"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Управление</string>
    </property>
    <addaction name="action_6"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Открыть для чтения</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Клавиши</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
